site/src/sphinx/server-annotated-service.rst

```diff
@@ -455,7 +479,7 @@ converting ``MyRequest``.
 Injecting value of parameters and HTTP headers into a Java object
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-:api:`BeanRequestConverterFunction` is a built-in request converter for Java object. You can use it by
+There is a built-in request converter for Java object. You can use it by
```

Comment:
- There is a built-in -> Armeria provides a generic built-in
- Java object. You can use it by -> Java objects which is activated by